docs/agent-api.asciidoc

```diff
@@ -560,8 +561,8 @@ Use the <<api-request-size,`apiRequestSize`>> config option to control the byte
 [[api-request-size]]
 ===== `apiRequestSize`
 
-* *Type:* Number
-* *Default:* `1048576` (1 MiB)
+* *Type:* String
+* *Default:* `1mb` (1 MiB)
```

Comment:
No reason to write the MB size in brackets anymore

Reply:
fixed